webapp-landing/pom.xml

```diff
@@ -37,6 +37,16 @@
             <artifactId>jackson-databind</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.5</version>
+        </dependency>
+        <dependency>
```

Comment:
service-base module in Oskari includes the org.json classes. Remove this dependency to prevent any version conflicts.

Reply:
Nevermind, forgot that this is a separate webapp and doesn't include any Oskari code :)